CMakeLists.txt

```diff
@@ -476,6 +476,7 @@ add_subdirectory(mem.so)
 add_subdirectory(babel.so)
 add_subdirectory(perf.so)
 add_subdirectory(tst.so)
+add_subdirectory(runexternal.so)
```

Comment:
Can we think of better name for this module to support external components ? Like tpm.so/tpc.so (third party module/component) or em.so (external module) ... ?